draft-ietf-httpbis-digest-headers.md

```diff
@@ -1059,11 +1064,32 @@ in conjuction with the encrypted content-coding {{?RFC8188}}.
 
 ## Algorithm Agility
 
-...
+The security properties of digest-algorithms are not fixed.
+Algorithm agility is achieved establishing the IANA Digest Algorithm Values
+registry (see {{iana-digest-algorithm-registry}}) and Section 2.1 of {{?RFC7696}}).
+
+To help endpoints make secure algorithm choices,
+this document adds to the IANA Digest Algorithm Values registry
+a new "Status" field containing the most-recent appraisal of the digest-algorithm according
+to the algorithm used to compute the checksum based on the type and status of the primary document
+in which the checksum algorithm is defined.
```

Comment:
The remember of the paragraph is hard to parse and I don't think it is strictly true. I'd like to think that we can deprecate algorithms without the need of coordination with the original defining document. The reason some algorithm becomes deprecated or obsolete could because it is a dead. I think it would be simpler to just remove everything after "... appraisal of the digest-algorithm."

Reply:
In our spec  https://github.com/httpwg/http-extensions/blame/master/draft-ietf-httpbis-digest-headers.md#L1079:1082
we reused this part of RFC3864 

      Specify "standard", "experimental", "informational", "historic",
      "obsoleted", or some other appropriate value according to the type
      and status of the primary document in which it is defined.  For
      non-IETF specifications, those formally approved by other
      standards bodies should be labelled as "standard"; others may be
      "informational" or "deprecated" depending on the reason for
      registration.

which is more relaxed than the present. I will relax our statement and ask to reviewers for feedback so that we can land this PR.

Reply:
```suggestion
this document adds to the IANA Digest Algorithm Values registry
a new "Status" field containing the most-recent appraisal of the digest-algorithm.
```